test/random.jl

```diff
@@ -313,7 +313,9 @@ for rng in ([], [MersenneTwister(0)], [RandomDevice()])
                    (Set(rand(Int, 20)), Int),
                    (Dict(zip(rand(Int,10), rand(Int, 10))), Pair{Int,Int}),
                    (1:100, Int),
-                   (rand(Int, 100), Int)]
+                   (rand(Int, 100), Int),
+                   ("qwèrtï", Char),
+                   (Base.Test.GenericString("qwèrtï"), Char)]
```

Comment:
I would throw in a test for empty strings since that seems like a corner case that could easily break. (I checked this out specifically to test that – it works.)